PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs

```diff
@@ -11,7 +11,7 @@
 using System.Windows.Input;
```

Comment:
Type `this.` and then you can access the PowerPoint context provided by Action Framework.